src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/PackagePropertyPage/AssemblyVersionValueProvider.cs

```diff
@@ -22,7 +22,7 @@ internal sealed class AssemblyVersionValueProvider : BaseVersionValueProvider
                 return s_DefaultAssemblyVersion;
             }
 
-            return new Version(defaultVersion.Major, defaultVersion.Minor, defaultVersion.Build, revision: 0);
+            return new Version(defaultVersion.Major, defaultVersion.Minor, Math.Max(defaultVersion.Build, 0), revision: Math.Max(defaultVersion.Revision, 0));
```

Comment:
Help me understand why VS is computing the default version of AssemblyVersion? Given that it doesn't run during build...